webcompat/webhooks/helpers.py

```diff
@@ -49,7 +51,10 @@ def set_label(label, issue_number):
     api_post('labels', payload, issue_number)
 
 
-def dump_to_db(title, body, issue_number):
+def dump_to_db(title, body, issue_number, status, reported_from, creation_time, last_change_time):
```

Comment:
line too long.
You really need to install a Python linter for your code editor. These are very simple syntactic issues.